### ISSUE TITLE
why Limit to neuroscience ?

### ISSUE THREAD

**Issue by jcolomb** [INCF/neuroscience-data-structure#6]
Is there any reason to restrict the scope of this project to "neuroscience" ? Couldn't we think about life science in general?

Especially when one consider that neuroscientists sometimes also do metabolics studies, immunology studies, molecular analysis, single cell proteomics,... They may want to keep the same data structure for all their experiments ?

**Comment by SylvainTakerkart** [comment]
indeed, this is a question we should ask ourselves, and in general: "what is the scope and aim of this data structure?"

our reasoning has been that there is already a large-enough diversity in neuroscience so that our endeavor might be difficult... my personal feeling is that maybe we might have to restrict ourselves to a subpart of neuroscience at first, while thinking about keeping things as flexible as possible so that extensions to other subparts of neuroscience (or other fields in life science) remain easy... anyhow let's see how the other issues progress (the three main ones listed below) and we will clearly reconsider this question later on...

#2 
#3 
#4

**Comment by yarikoptic** [comment]
the overall approach should indeed have minimal set of "assumptions" (e.g. "having a participant level directory") and thus possibly generalize across fields of endeavor.  Standardization of fields naming cannot avoid to be domain specific but ideally should indeed have terms possibly used in other  fields in mind to avoid possible future ambiguity etc. (e.g. vague terms like "imaging" should be avoided etc).  Values should follow some standard formats (e.g. ISO 8601 for date time) allowing for arbitrary precision which might be needed in specific cases (e.g. subsecond in time), and see https://bids-specification.readthedocs.io/en/stable/99-appendices/05-units.html#unit-table for units standards used in BIDS.

**Comment by SDV96** [comment]
IS THIS ISSUE SOLVED I CAN IMPROVE IT By..... if @jcolomb   approves  assign within 1 WEEKS {WORKING NOW}

**Comment by yarikoptic** [comment]
Overall, it has already expanded outside of neuroscience as supports purely behavioral datasets and WiP such as

- https://github.com/bids-standard/bids-specification/pull/2231

have a look. @SDV96 what are your use cases?

**Comment by SDV96** [comment]
@yarikoptic i have completed some work and having some issues  will figure out
UNTIL assign me FOR THIS ISSUE WILL COMPLETE UNDER 1 WEEK

**Comment by SylvainTakerkart** [comment]
most of  the issues on this repo were mostly used as a discussion forum around specific topics, that we then addressed in the BEP... I will close them to avoid confusion, and modify the readme file at the root of the repo...
